src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,103 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+const styles = {
+  fontFamily({ theme, _weight }) {
+    return theme.fonts.family.lato[_weight];
+  },
+  fontSize({ theme, size }) {
+    return theme.fonts.size[size];
+  },
+  color({ theme, color }) {
+    return getColor(theme, color);
+  },
+  textDecorationLine({ _isUnderlined }) {
+    if (_isUnderlined) {
+      return 'underline';
+    } else {
+      return 'none';
+    }
+  },
+  align({ align }) {
+    switch (align) {
+      case 'left':
+      default:
+        return 'flex-start';
+      case 'center':
+        return 'center';
+      case 'right':
+        return 'flex-end';
+    }
+  },
+  letterSpacing({ theme, _letterSpacing }) {
+    return `${theme.fonts.letterSpacing[_letterSpacing]}`;
+  },
+  lineHeight({ theme, size, _lineHeight }) {
+    if (_lineHeight) {
+      return theme.fonts.lineHeight[_lineHeight];
+    } else {
+      return getLineHeight(theme, size);
+    }
+  },
+  overflow({ truncate }) {
+    if (truncate) {
+      return 'hidden';
+    } else {
+      return '';
+    }
+  },
+  whiteSpace({ truncate }) {
+    if (truncate) {
+      return 'nowrap';
+    } else {
+      return '';
+    }
+  },
+  textOverflow({ truncate }) {
+    if (truncate) {
+      return 'ellipsis';
+    } else {
+      return '';
+    }
+  },
+};
+
+const Text = styled.div`
+  font-family: ${styles.fontFamily};
+  font-size: ${styles.fontSize};
+  color: ${styles.color};
+  text-decoration-line: ${styles.textDecorationLine};
+  align-self: ${styles.align};
+  letter-spacing: ${styles.letterSpacing};
+  line-height: ${styles.lineHeight};
+  overflow: ${styles.overflow};
```

Comment:
For single ternary conditions let's stop avoiding use of styles. it seems like overdoing 
```suggestion
  overflow: ${(props) => props.truncate ? 'hidden' : ''};
```

Reply:
done